Header header
duration cam_frame_delay
float32[30] left
float32[30] right
